pyproject.toml

```
@@ -192,3 +194,19 @@ strict = true
 [tool.codespell]
 skip = ".git,*.pdf,*.svg"
 ignore-words-list = "theis,coo,homogenous"
+
+[tool.towncrier]
+package = "anndata"
+directory = "docs/release-notes"
+filename = "docs/release-notes/{version}.md"
+single_file = false
+package_dir = "src"
+issue_format = "{{issue}}`{issue}`"
```

Since we actually link PRs and not issues:

```suggestion
issue_format = "{{pr}}`{issue}`"
```